include/dlaf/eigensolver/bt_reduction_to_band/impl.h

```diff
@@ -60,7 +60,7 @@ template <>
 struct Helpers<Backend::GPU> {
   template <class T>
   static void copyAndSetHHUpperTiles(SizeType j_diag, const matrix::Tile<const T, Device::GPU>& src,
-                                     matrix::Tile<T, Device::GPU>& dst, cudaStream_t stream) {
+                                     const matrix::Tile<T, Device::GPU>& dst, cudaStream_t stream) {
```

Comment:
This did not compile with a non-const reference. I will check separately why that happens. I've for now changed it to a const reference (which is what we use in other tile operations) and I'm tracking it here: https://github.com/pika-org/pika/issues/136.